server/ecore-glsp/src/main/java/com/eclipsesource/glsp/ecore/EcoreServerLauncher.java

```diff
@@ -26,7 +28,7 @@ public static void main(String[] args) {
 		EcorePackage.eINSTANCE.eClass();
 		ElkLayoutEngine.initialize(new LayeredMetaDataProvider());
 		BasicConfigurator.configure();
-		
+		Logger.getRootLogger().setLevel(Level.DEBUG);
```

Comment:
do we need this?

Reply:
This was added by @tortmayr.

Reply:
Most of the glsp logging on server side is set to the `Debug` Level but `BasicConfiguration.configure()` sets the Log level to `Info` which hides lots of usefull log messages

Reply:
Thank you for clarifying